setup.py

```diff
@@ -49,6 +49,7 @@
     "tablib>=0.12.1,<0.13",
     "werkzeug>=0.14.1 ; python_version<'3.10'",
     "werkzeug>=0.15.0 ; python_version>='3.10'",
+    "swagger_spec_validator<3.0.0",  # 3.0.0 drops support for python 3.6
```

Comment:
Perhaps you can do the `; python_version<'3.7'"`  constraint here, so that later swagger versions would still be installed for py37 and above? (Could be useful for swagger bug fixes etc.)